test/unit/authentication/authentication_test.js

```diff
@@ -160,7 +160,7 @@ describe('external browser authentication', function ()
     webbrowser = require('webbrowser');
     httpclient = require('httpclient');
 
-    var auth = new auth_web('', webbrowser.open, httpclient);
+    var auth = new auth_web('', webbrowser.open, httpclient, -10000);
```

Comment:
should it crash? the crash reason will be the missing url or negative timeout?

Reply:
fixed